src/components/Icon/IconMore.test.js

```diff
@@ -0,0 +1,10 @@
+/* global describe, expect, it, shallow */
+import React from 'react';
+import IconArrow from './IconMore';
+
+describe('IconMore', () => {
+    it('has the appropriate classes', () => {
```

Comment:
Can we have a test to verify `render()` returns SVG content? (see https://github.com/Mudano/ui-react/pull/47#discussion_r166914332). We probably need to add these tests for each icon component.

Reply:
Test added.
I've also created a helper for that which I'll include in a separate PR.

Reply:
👍